#include "dialog_vulndetailinfo.h"
#include "ui_dialog_vulndetailinfo.h"

Dialog_VulnDetailInfo::Dialog_VulnDetailInfo(QWidget *parent) :
    QDialog(parent),
    ui(new Ui::Dialog_VulnDetailInfo)
{
    ui->setupUi(this);
}

Dialog_VulnDetailInfo::~Dialog_VulnDetailInfo()
{
    delete ui;
}

void Dialog_VulnDetailInfo::set_vulnInfoStruct(VulnDetailInfo vulnDetailInfo)
{
    ui->label_vulnName_v->setText(vulnDetailInfo.name);
    ui->label_vulnType_v->setText(vulnDetailInfo.type);
    ui->label_vulnRank_v->setText(vulnDetailInfo.rank);
    ui->label_vulnTime_v->setText(vulnDetailInfo.time);
    ui->label_vulnEffect_v->setText(vulnDetailInfo.effect);
    ui->label_vulnCategory_v->setText(vulnDetailInfo.category);
    ui->label_vulnCve_v->setText(vulnDetailInfo.cve);
    ui->label_vulnDork_v->setText(vulnDetailInfo.dork);
    ui->label_vulnCase_v->setText(vulnDetailInfo.case_);
    ui->label_vulnRefer_v->setText(vulnDetailInfo.refer);
    ui->label_vulnDesc_v->setText(vulnDetailInfo.desc);
    ui->label_vulnVerify_v->setText(vulnDetailInfo.verify);
    ui->label_vulnRepair_v->setText(vulnDetailInfo.repair);
    ui->label_vulnPayload_v->setText(vulnDetailInfo.payload);
    ui->label_vulnKeyword_v->setText(vulnDetailInfo.keyword);
}
